src/confluent_kafka/avro/__init__.py

```diff
@@ -108,13 +107,13 @@ def produce(self, **kwargs):
             else:
                 raise ValueSerializerError("Avro schema required for values")
 
-        if key is not None:
+        if key is not None:            
```

Comment:
```suggestion
        if key is not None:
```

Reply:
There's a lot of issues like this, and I'd mostly filed them under "get black fixed". See https://github.com/confluentinc/confluent-kafka-python/pull/1352 and https://github.com/confluentinc/confluent-kafka-python/pull/1493#issuecomment-1479400389

Reply:
Ah yes makes sense. Unfortunately maintainers don't seem pretty active

Reply:
Sadly, yes. Ironically, I'm job hunting at the moment, and had applied to Confluent but they were looking mostly for Java devs IIRC :( If they wanted to employ me to fix these things that might speed things up a lot...

Reply:
> If they wanted to employ me to fix these things that might speed things up a lot...

Oh please ! This project is half-dead and Confluent is a commercial entity.